dev/com.ibm.ws.session.cache/resources/com/ibm/ws/session/store/cache/resources/WASSessionCache.nlsprops

```diff
@@ -19,4 +19,32 @@
 
 INCORRECT_URI_SYNTAX=SESN0300E: Invalid syntax in httpSessionCache URI.  Cause is {0}.
 INCORRECT_URI_SYNTAX.explanation=The syntax of the specified httpSessionCache JCache configuration URI is invalid.
-INCORRECT_URI_SYNTAX.useraction=Correct the syntax of the specified httpSessionCache configuration URI.
\ No newline at end of file
+INCORRECT_URI_SYNTAX.useraction=Correct the syntax of the specified httpSessionCache configuration URI.
+
+ERROR_REMOVING_SESSION=SESN0301E: An error occurred when trying to remove a session from the cache. The exception is: {0}.
+ERROR_REMOVING_SESSION.explanation=An error occurred when trying to remove a session from the cache. See exception for further details.
+ERROR_REMOVING_SESSION.useraction=See the exception for more details. The JCache provider's configuration may need to be updated.
+
+PROP_HIT_ERROR=SESN0302E: An exception occurred when storing application data changes to the cache. The exception is: {0}.
+PROP_HIT_ERROR.explanation=An exception occurred when storing application data changes to the cache. See exception for further details.
+PROP_HIT_ERROR.useraction=See the exception for more details. The JCache provider's configuration may need to be updated.
+
+STORE_SESS_ERROR=SESN0303E: An exception occurred when trying to store a session in the cache. The exception is: {0}.
+STORE_SESS_ERROR.explanation=An error occurred when trying to store a session in the cache. See exception for further details.
+STORE_SESS_ERROR.useraction=See the exception for more details. The JCache provider's configuration may need to be updated.
+
+LOAD_VALUE_ERROR=SESN0304E: An exception occurred when trying to read in an object of the application data for a session from the cache. The exception is: {0}.
+LOAD_VALUE_ERROR.explanation=An error occurred when trying to read in an object of the application data for a session from the cache. See exception for further details.
+LOAD_VALUE_ERROR.useraction=See the exception for more details. The JCache provider's configuration may need to be updated.
+
+ERROR_CACHE_ACCESS=SESN0305E: An exception occurred while attempting to access the cache. The exception is: {0}.
+ERROR_CACHE_ACCESS.explanation=An exception has been detected while attempting to access the database. See exception for further details.
+ERROR_CACHE_ACCESS.useraction=See the exception for more details. The JCache provider's configuration may need to be updated.
+
+ERROR_SESSION_INVAL=SESN0306E: An exception occurred when trying to invalidate a session in the cache. The exception is: {0}.
+ERROR_SESSION_INVAL.explanation=An error occurred when trying to invalidate a session in the database. See exception for further details.
+ERROR_SESSION_INVAL.useraction=See the exception for more details. The JCache provider's configuration may need to be updated.
+
+ERROR_SESSION_INIT=SESN0307E: An exception occurred when trying to initialize the cache. Exception is: {0}.
```

Comment:
This message is different from the others in that it states "Exception is" rather than "The exception is".  Can we make them all consistent?